save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/testsuitessources/fetch/TestSuitesSourceFetcher.kt

```diff
@@ -200,9 +200,10 @@ private fun innerTestSuitesSourceFetcher() = FC<TestSuitesSourceFetcherProps> {
                         headers = jsonHeaders,
                         loadingHandler = ::loadingHandler,
                     )
-                        .unsafeMap {
+                        .unsafeMap<List<String>> {
                             it.decodeFromJsonString()
                         }
+                        .also { setSelectedValue("") }
```

Comment:
it's nullable -- maybe `seSelectedValue(null)`

Reply:
Done